modules/installation-configuration-parameters.adoc

```diff
@@ -1101,6 +1105,10 @@ Additional Azure configuration parameters are described in the following table:
 |Enables the use of Azure ultra disks for persistent storage on control plane machines. This requires that your Azure region and zone have ultra disks available.
 |`Enabled`, `Disabled`. The default is `Disabled`.
 
+|`controlPlane.platform.azure.vmNetworkingType`
+|Enables accelerated networking. Accelerated networking enables single root I/O virtualization (SR-IOV) to a VM, improving its networking performance. If instance type of control plane machines support `Accelerated` networking, installer by default enables `Accelerated` networking, otherwise the default networking type is `Basic`.
```

Comment:
```suggestion
|Enables accelerated networking. Accelerated networking enables single root I/O virtualization (SR-IOV) to a VM, improving its networking performance. If instance type of control plane machines support `Accelerated` networking, by default, the installer enables `Accelerated` networking, otherwise the default networking type is `Basic`.
```

Reply:
Also the `If instance type of compute machines support...` statement is mentioned in `Values` column for the other 2 fields. can you make this consistent?